cmd/cli/options/root.go

```diff
@@ -0,0 +1,19 @@
+package options
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// RootOptions defines the options for the `root` command
+type RootOptions struct {
+	ConfigPath string
+	Verbose    bool
+	Debug      bool
+}
+
+// AddFlags adds command line flags for the RootOptions struct
+func (ro *RootOptions) AddFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVarP(&ro.ConfigPath, "config", "c", "", "path to config file")
+	cmd.PersistentFlags().BoolVarP(&ro.Verbose, "verbose", "v", false, "verbose output")
+	cmd.PersistentFlags().BoolVar(&ro.Debug, "debug", false, "debug output")
```

Comment:
can we add a `level` flag or at maybe support `-vv`.

Reply:
sure, we can't easily make debug with -vv . btw only place i've seen the usage of -vv (or -vvv) is with syft :) but ill add that and remove the --debug flag so we have -v (info) -vv (debug) -vvv (trace, if supported)